src/effects/effectknobparameterslot.cpp

```diff
@@ -171,7 +182,7 @@ void EffectParameterSlot::onEffectMetaParameterChanged(double parameter, bool fo
                     return;
                 }
                 {
```

Comment:
There is no need to have this in its own block. Delete the unnecessary block and unindent the following code.

Reply:
It is their in it's own block because of the local variable declaration of `double neutral` inside the switch case. I've kept it as it is for now. Otherwise we can declare `neutral` outside of the switch case statement and remove the block.

Reply:
Why does that variable need its own block scope though? It doesn't conflict with anything else in the function's scope, does it?